content/francineblanc/how-to-use-git-for-beginners.md

```diff
@@ -0,0 +1,219 @@
+
+![git-status-untracked](git-status-untracked-files.png)
+
+You will see the output says ```On branch main``` which you can ignore at the moment. You'll also see a mysterious file called ```.DS_Store``` which is a file that is automatically created by Mac OS X. This file contains information about system configurations, so should not be committed as part of the Git workflow. To make sure this file is not added to the commit, a special file called ```.gitignore``` must be made in the Working Directory.
+
+### .gitignore
+The ```.gitignore``` file is a special file which generally lives in the root of the Working Directory. It contains a list of files and/or directories which a developer has added that Git should not track, and so will not be committed. These files may be log files, or particular modules in a project that shouldn't be tracked, or (for Mac users) files like ```.DS_Store```. Make sure to try and remember to add files which should not be tracked ***before*** they are committed.
+
+Now, with the ```.gitignore file``` made and the ```.DS_Store``` file added, running ```git status``` in the terminal shows the following output:
+
+![git-status-untracked-with-gitignore](git-status-untracked-with-ignore.png)
+
+## Making Changes and Adding Files to the Staging Area
+In order for Git to start tracking changes, files need to be added to the Staging Area. This can be done with the command:
+```
+git add <filename>
+```
+where filename is the name and extension of the file to be added to the staging area. Multiple files can also be added with the syntax:
+
+```
+git add <filename1> <filename2> <filename3>
+```
+
+If all files in the Working Directory need to be tracked, the syntax
+
+```
+git add .
+```
+can be used.
+
+Once a file has been added, ```git status``` can be run in the terminal/command prompt to check that the file is in the Staging Area. Git will show the changed files to be committed in <span style="color:green">green</span> text like so:
+
+![git-status-untracked-with-gitignore](git-add-and-status.png)
+
+You can find out more about the additional extensions to ```git add``` in the [Add] section of the Codecademy Docs for Git.
+
+### Oops! I didn't want to add that!
+There may be occasions where a file has been incorrectly added to the Staging Area. This can be undone with the command ```git reset <filename>```. This will not affect the changes done to the file in any way, it will simply remove it from the staging area. 
+
+In the case where all files needs to be unstaged, the command ```git reset``` can be run without adding further arguments.
+
+Note that as of ```git version 2.24``` there is alternative syntax to unstage files, which is ```git restore --staged <filename>``` where ```filename``` is the name of the file to be unstaged.
```

Comment:
```suggestion
As of version 2.24, there is alternative syntax for unstaging files in Git, which is ```git restore --staged <filename>``` where ```filename``` is the name of the file to be unstaged.
```

For the following latter part of this text:

> which is ```git restore --staged <filename>``` where ```filename``` is the name of the file to be unstaged

I would break up in the following way (and use \```pseudo for the snippet):

As of version 2.24, there is alternative syntax for unstaging files in Git:

```pseudo
git restore --staged <filename>
```

The `<filename>` is the name of the file to be unstaged.